src/common/tools/testTranslators/testTranslators.html

```diff
@@ -27,11 +27,10 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 	<head>
 		<script type="text/javascript" src="/browser-polyfill.js"></script>
-		<script type="text/javascript" src="/node_modules.js"></script>
 		<script type="text/javascript" src="/zotero_config.js"></script>
 		<script type="text/javascript" src="/zotero.js"></script>
 		<script type="text/javascript" src="/translate/promise.js"></script>
-		<script type="text/javascript" src="/translate/resource/zoteroTypeSchemaData.js"></script>
+		<script type="text/javascript" src="/utilities/resource/zoteroTypeSchemaData.js"></script>
```

Comment:
I've checked out zotero-schema directly into this repo now. You can load it at `/zotero-schema/zoteroTypeSchemaData.js`. Now when building the connector there's no need for a recursive submodule checkout.

Reply:
Do you mean `/zotero-schema/schema.json`?

Reply:
But also, isn't this about the build files, not the source files? `/zotero-schema` doesn't exist in the build, as far as I can see. Isn't this data just in `/schema.js`?

Reply:
Ah yes, you're right. It is just `/schema.js`